### PR TITLE
Corrected spelling errors in Event Faqs

### PR DESCRIPTION
**Pull request by katchoua**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -469,7 +469,7 @@ <h4 class="panel-title">
...
+                                	<p> SMConf 2016 will be held June 18, 2016 at Chariot Hotel Complex in Mile 18, Buea. The Chariot Hotel Complex is located 1 mile from Mile 17 in Buea and 70.1 km from the Douala International Airport.For information on driving directions and garage parking click<a href="http://bit.ly/1Sby1eI" style="color: #4F742A; font-weight: bolder;"> here</a>. The conference will include a full day divided between general sessions in the morning and three different breakout sessions in the afternoon: Design Studios, Hackers Sandbox and Community Pavilion. We will also have a Startup booth after Lunch for Developers and Designers to have one-on-one sessions with Startups. </p>
                                 </div>
```
Comment (ch3ck) [inline index.html:473]: What have you changed here? I don't see anything different @katchoua

File: index.html [hunk 2]
```diff
@@ -495,7 +495,7 @@ <h4 class="panel-title">
 
                             <div id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour">
                                 <div class="panel-body">To register for SMConf 2016, please visit the registration page. <br><b>How much does it cost to attend?</b><br> SMConf 2016 is free, but tickets are limited to 500 attendees only.
-                                <br> However Startups requesting booths are required to register to due space limits.
+                                <br> However, Startups requesting booths are required to register to due space limits.
                                 <br> Buea Startup Registration: XAF 40,000($80)
```
Comment (ch3ck) [inline index.html:499]: This looks good though

**Comment by katchoua** [comment]
corrected this: three difference breakout sessions..., to this: three different breakout sessions
and this: However Startups to this: However, Startups...

**Comment by ch3ck** [comment]
Ahh Cool. Nice work! @katchoua

**Pull request merged**